databricks/koalas/indexes.py

```diff
@@ -1961,6 +1961,31 @@ def holds_integer(self):
         """
         return isinstance(self.spark.data_type, IntegralType)
 
+    def item(self):
+        """
+        Return the first element of the underlying data as a python scalar.
+
+        Returns
+        -------
+        scalar
+            The first element of Index.
+
+        Raises
+        ------
+        ValueError
+            If the data is not length-1.
+
+        Examples
+        --------
+        >>> kidx = ks.Index([10])
+        >>> kidx.item()
+        10
+        """
+        result = self._internal.spark_frame.head(2)
```

Comment:
I think you can call `self.head(2).to_pandas` instead of `head`.

Reply:
or `self.to_series.item()`

Reply:
Okay, `self.to_series().item()` seems fine because `Index` has no `head`

Thanks!!